Harmony/Tools/AccessTools.cs

```diff
@@ -1703,9 +1715,6 @@ public static object MakeDeepCopy(object source, Type resultType, Func<string, T
 					}
 					return addableResult;
 				}
-
-				// TODO: add dictionaries support
```

Comment:
Does this already handle dictionaries then via `Add(KeyValuePair<...>)`?

Reply:
that's a legacy comment from Pardeike